dropwizard-jersey/src/test/java/io/dropwizard/jersey/jackson/JsonProcessingExceptionMapperTest.java

```diff
@@ -97,7 +97,10 @@ public void returnsA400ForNonDeserializableRequestEntities() throws Exception {
 
     @Test
     public void returnsA400ForWrongInputType() throws Exception {
```

Comment:
Gah, that's because the exception message starts with "No suitable constructor", which is [one of our heuristics](https://github.com/dropwizard/dropwizard/blob/0880fcdfd1c37e2343c362ba7b23b2d14ff09d8e/dropwizard-jersey/src/main/java/io/dropwizard/jersey/jackson/JsonProcessingExceptionMapper.java#L56-L61) for a programmer error :crying_cat_face: 

We'll have to change the heuristics (and not the test case).